editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/util/maven/MavenUtils.java

```diff
@@ -91,7 +91,7 @@ public static void updateCamelVersionDependencies(Model mavenModel, List<Depende
 		if(camelVersion.contains(REDHAT_NAMING_USED_IN_VERSION)){
 			for (Dependency dep : dependencies) {
 				if (isCamelDependency(dep)) {
-					if(isMavenPropertyFuseBomVersionSet(properties)){
+					if(isMavenPropertyFuseBomVersionSet(properties) && !CamelModelFactory.isPureFISVersion(camelVersion)){
```

Comment:
it is because we didn't implemented a way to support different BOMs, there is a different ones for Fuse and FIS. To have something working, I created https://issues.jboss.org/browse/FUSETOOLS-2242 to support it later on.
As currently we have a single working FIS version, it is working, it will be required to implement the FUSETOOLS-2242 when we will support different versions of FIS.